### DISCUSSION TITLE
nextcloud container unable to reach itself using domain.tlk

### DISCUSSION THREAD

**Discussion by storm1er** [nextcloud/all-in-one#1815] [Questions] [answered]
### Steps to reproduce

I noticed some logs in my nextcloud container which wasn't here before removing my reverse proxy:
![image](https://user-images.githubusercontent.com/9821946/213706859-6c9a6cac-b90e-44aa-85c4-4b07c15e4ae4.png)

So I did a test:
- Connect to my RPI
- ping domain.tld => OK
- connect to the nextcloud container
- ping domain.tld => KO

### Expected behavior 

Container should be able to ping his domain.tld

### Actual behavior 

![image](https://user-images.githubusercontent.com/9821946/213706667-99bd847f-59f5-4daa-a9fa-56972fc2ccf4.png)

### Host OS 

RPi 4B, debian

#### Nextcloud AIO version 

4.1.0

#### Current channel 

latest-arm64

#### Other valuable info 

How did I remove the reverse proxy:
- Moved DNS record to my CNAME dynDns
- Open/Redirect ports 80, 443, 8443
- Open AIO
- "Stopped containers"
- Removed APACHE_PORT variable
- Removed SKIP_DOMAIN_CHECK variable
- Readded ports which was flag as "Can be removed if using a reverse proxy"
- Restarted mastercontainer
- "Start containers"

using domain.tld from my phone on 4G works well, cert has changed, traceroute has changed and does not go through Cloudflare anymore (which I was using as a reverse proxy)

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?
